modules/swagger-models/src/main/java/io/swagger/models/refs/RefType.java

```diff
@@ -0,0 +1,9 @@
+package io.swagger.models.refs;
+
+/**
+ * Created by russellb337 on 7/1/15.
+ */
+public enum RefType {
+    DEFINITION,
+    PARAMETER
+}
```

Comment:
To be fully in line with in line with https://github.com/swagger-api/swagger-spec/blob/master/guidelines/REUSE.md - shouldn't we have OPERATION and RESPONSE refs as well (with corresponding changes in the other classes) ?